src/digitalmarketplace/components/banner/template.njk

```diff
@@ -1,5 +1,6 @@
-<section
+<div
```

Comment:
Good spot about the tag-mismatch here.

Is there a reason to not prefer section?

Reply:
<section> the better choice here - I was going by the Design System Error Summary - wasn't aware of the banner component you shared!